worlds/mlss/docs/setup_en.md

```diff
@@ -0,0 +1,53 @@
+# Setup Guide for Mario & Luigi: Superstar Saga Archipelago
+
+## Important
+
+As we are using Bizhawk, this guide is only applicable to Windows and Linux systems.
+
+## Required Software
+
+- Bizhawk: [Bizhawk Releases from TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+  - Version 2.9.1 is recommended.
+  - Detailed installation instructions for Bizhawk can be found at the above link.
+  - Windows users must run the prereq installer first, which can also be found at the above link.
+- The built-in Bizhawk client, which can be installed [here](https://github.com/ArchipelagoMW/Archipelago/releases)
+- A US Mario & Luigi: Superstar Saga ROM
+
+## Optional Software
+
+- [Poptracker](https://github.com/black-sliver/PopTracker/releases)
+  -[MLSS Autotracker](https://github.com/seto10987/MLSS-PopTracker/releases)
```

Comment:
```suggestion
  - [MLSS Autotracker](https://github.com/seto10987/MLSS-PopTracker/releases)
```
I think missing the space here breaks on webhost? Best to test that if you haven't already